spec/presenters/organisations_presenter_spec.rb

```diff
@@ -196,46 +177,48 @@
         ],
       ]
 
-      assert_equal expected, @organisations_presenter.ordered_works_with(test_org)
+      expect(organisations_presenter.ordered_works_with(test_org)).to eq(expected)
     end
   end
 
   describe "organisation type" do
-    before :each do
-      content_item = ContentItem.new(ministerial_departments_hash)
-      content_store_organisations = ContentStoreOrganisations.new(content_item)
-      @organisations_presenter = Organisations::IndexPresenter.new(content_store_organisations)
-    end
+    let(:organisations_presenter) { presenter_from_hash(ministerial_departments_hash) }
 
     it "executive_office? returns true if organisation is number_10" do
       test_organisation_type = :number_10
-      assert @organisations_presenter.executive_office?(test_organisation_type)
+      expect(organisations_presenter.executive_office?(test_organisation_type)).to be
```

Comment:
one of the weird `.to be`'s slipped through your net

Reply:
same comment 203, 209

Reply:
👍